app/factories/common.py

```diff
@@ -7,10 +8,10 @@
 integer_ = 25
 url_ = static_page
 imageUrl_ = image_link
-date_ = '2016-12-13T23:59:59.123456+00:00'
-dateFuture_ = '2099-12-13T23:59:59.123456+00:00'
-dateEndFuture_ = '2099-12-14T23:59:59.123456+00:00'
-dateEnd_ = '2020-12-14T23:59:59.123456+00:00'
+date_ = datetime.datetime(2016, 12, 13)
```

Comment:
Why was this change needed?

Reply:
![Screenshot from 2019-03-21 03-09-33](https://user-images.githubusercontent.com/25197147/54700393-f4eeb280-4b58-11e9-85a1-24f47b561591.png)